cg/apps/demultiplex/sample_sheet/sample_sheet_creator.py

```diff
@@ -234,21 +241,21 @@ def get_additional_sections_sample_sheet(self) -> list[list[str]]:
             SampleSheetBCLConvertSections.Header.INDEX_ORIENTATION_FORWARD.value,
         ]
         reads_section: list[list[str]] = [
-            [SampleSheetBCLConvertSections.Reads.HEADER.value],
+            [SampleSheetBCLConvertSections.Reads.HEADER],
             [
-                SampleSheetBCLConvertSections.Reads.READ_CYCLES_1.value,
+                SampleSheetBCLConvertSections.Reads.READ_CYCLES_1,
                 self.run_parameters.get_read_1_cycles(),
             ],
             [
-                SampleSheetBCLConvertSections.Reads.READ_CYCLES_2.value,
+                SampleSheetBCLConvertSections.Reads.READ_CYCLES_2,
                 self.run_parameters.get_read_2_cycles(),
             ],
             [
-                SampleSheetBCLConvertSections.Reads.INDEX_CYCLES_1.value,
+                SampleSheetBCLConvertSections.Reads.INDEX_CYCLES_1,
                 self.run_parameters.get_index_1_cycles(),
             ],
             [
-                SampleSheetBCLConvertSections.Reads.INDEX_CYCLES_2.value,
+                SampleSheetBCLConvertSections.Reads.INDEX_CYCLES_2,
```

Comment:
yes, tha dataclasses are defined differently. In these cases, the class `Reads` is a `StrEnum`, while for example `Settings` and `Header` are `Enum`. This is because for those classes the values can also be lists. If the dataclass is `StrEnum` you don't need the `.value`, if it is `Enum`, not adding `.value` will make Pycharm raise warnings, but I think it would still work